### PR TITLE
Order list

### PR DESCRIPTION
**Pull request by scresh**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -2,7 +2,8 @@ import {NgModule} from '@angular/core';
 import {Routes, RouterModule} from '@angular/router';
 
 const routes: Routes = [
-  {path: '', loadChildren: './tabs/tabs.module#TabsPageModule'}
+  {path: '', loadChildren: './tabs/tabs.module#TabsPageModule'},
+  { path: 'orders', loadChildren: './orders/orders.module#OrdersPageModule' }
```
Comment (Fiszcz) [inline src/app/app-routing.module.ts]: CTRL + ALT + L

File: src/app/orders/orders.module.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import { Routes, RouterModule } from '@angular/router';
+
+import { IonicModule } from '@ionic/angular';
+
+import { OrdersPage } from './orders.page';
+
+const routes: Routes = [
```
Comment (Fiszcz) [inline src/app/orders/orders.module.ts]: Is it necessary ? I think that you can remove this.

File: src/app/orders/orders.page.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+          }
+        },
+      ]
+    });
+
+    await actionSheet.present();
+  }
+
+
+getOrders() {
```
Comment (Fiszcz) [inline src/app/orders/orders.page.ts]: CTRL + ALT + L

File: src/app/orders/orders.page.ts [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    this.orderService.getOrders()
+      .then(data => {
+        this.orders = data;
+      });
+  }
+  pressEvent() {
```
Comment (Fiszcz) [inline src/app/orders/orders.page.ts]: add enter before this method. 
And remove console.log from the same method and the same action for tapEvent()

File: src/app/orders/orders.page.ts [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    this.getOrders();
+  }
+
+  async presentActionSheet() {
+    const actionSheet = await this.actionSheetCtrl.create({
+      buttons: [
+        {
+          text: 'Rate',
+          handler: () => {
+            console.log('Rate');
```
Comment (Fiszcz) [inline src/app/orders/orders.page.ts]: remove console.logs

**Comment by owlcode** [comment]
Up

File: src/app/orders/service/orders.service.ts [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+  constructor(public http: HttpClient) {
+  }
+
+  getOrders() {
+    return new Promise(resolve => {
+      this.http.get(this.apiUrl + '/orders').subscribe(data => {
+        resolve(data);
+      }, err => {
+        console.log(err);
```
Comment (Fiszcz) [inline src/app/orders/service/orders.service.ts]: For rejected situaction, you can save somewehere information about error and show simple information at order tab. 

You can create variable at here (OrdersService)  for failure and use it in order component to get info for user about problems.
Or you can use here resolve() with feedback to component with failure and then ...

File: src/app/orders/service/orders.service.ts [hunk 7]
```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class OrdersService {
+  apiUrl = 'http://localhost:3000/api';
```
Comment (Fiszcz) [inline src/app/orders/service/orders.service.ts]: Set temporaly: https://stark-sands-86101.herokuapp.com/api

Or better use interceptor.

**Pull request merged**